README.md

```diff
@@ -484,6 +485,12 @@ A curated list of awesome Java frameworks, libraries and software.
 - [ReflectASM](https://github.com/EsotericSoftware/reflectasm) - ReflectASM is a very small Java library that provides high performance reflection by using code generation.
 - [Reflections](https://github.com/ronmamo/reflections) - Reflections scans your classpath, indexes the metadata, allows you to query it on runtime and may save and collect that information for many modules within your project.
 
+### Job Scheduling
```

Comment:
Any more libraries? The CONTRIBUTE.md should indicate that we want to have new categories only if we have 3 or more entries.

Reply:
Added 3 more libraries.